Remove redundant entries from htaccess files

gosko opened the pull request:
W3C's site serves `ttl` files as `text/turtle`, `jsonld` files as `application/ld+json`, and `html` files under /ns/ as `utf-8` by default, so these config directives are no longer needed while they might have been in the past.

Also it sets `Access-Control-Allow-Origin` to `*` for many types including `text/turtle` and `application/ld+json` under `/ns/` so those directives are now redundant as well.

nigelmegitt reviewed (approved):
LGTM

gosko committed:
Convert HTML files to utf-8

gosko committed:
Remove redundant config directives

W3C's site serves ttl files as text/turtle, jsonld files as
application/ld+json, and html files under /ns/ as utf-8 by default, so
these config directives are no longer needed while they might have been
in the past.

gosko committed:
Remove more redundant config directives

vivienlacourba commented:
@gosko looks like there are a few more unneeded `AddType` directives to clean:

```bash
$ grep -r AddType
.htaccess:AddType application/n-triples .nt
lemon/.htaccess:#AddType application/rdf+xml .owl
credentials/.htaccess:AddType application/ld+json .jsonld
credentials/.htaccess:AddType text/turtle .ttl
credentials/status/.htaccess:AddType application/ld+json .jsonld
credentials/status/.htaccess:AddType text/turtle .ttl
credentials/json-schema/.htaccess:AddType application/ld+json .jsonld
```

gosko committed:
Remove more redundant config directives

vivienlacourba commented:
> @gosko looks like there are a few more unneeded `AddType` directives to clean:
> 
> ```shell
> $ grep -r AddType
> .htaccess:AddType application/n-triples .nt
> (...)
> ```

FYI n-triples is part of the default `mime.types` as confirmed on a W3C web server

```bash
# grep "application/n-triples" /etc/mime.types 
application/n-triples				nt
```

gosko commented:
> FYI n-triples is part of the default `mime.types` as confirmed on a W3C web server

Yes, I checked that for some others but missed this one. Thanks for reviewing!

gosko committed:
Replace unnecessary proxy rules with passthrough

gosko committed:
Fix broken proxying to a file in an archived wiki

gosko committed:
Add an archived copy of live-from-wiki.xsd

gosko committed:
Replace proxying of hydra files with local copies

core.jsonld and error.jsonld were copied from from the HydraCG repo,
https://github.com/HydraCG/Specifications/tree/master/spec/latest/core

gosko committed:
Remove expires and cache-control settings

This commit removes specific expiration and cache-control settings for
activitystreams-history since we manage these elsewhere.

gosko commented:
@vivienlacourba I think this is ready to merge, can you give it another quick review when you have a chance?

gosko committed:
Fix bug in hydra spec redirect

lanthaler commented on hydra/.htaccess:
```diff
@@ -0,0 +1,7 @@
+
+RewriteRule ^spec/(.*)$ https://www.hydra-cg.com/spec/$1 [R=302,L]
+RewriteRule ^context\.jsonld$ core [R,L]
+
```
As shared over mail already but for convenience here too:

It would be great to add the following here:

```
RewriteRule ^core/context\.jsonld$ core [R,L]

RewriteRule ^core$ core.jsonld [L,PT]
RewriteRule ^error$ error.jsonld [L,PT]
```

and make sure that

```
Header always set Access-Control-Allow-Origin "*"
```

is set

gosko replied:
Thanks! I added the redirect for core/context.jsonld to core.

The other rules should not be needed because our server has MultiViews enabled.

The ACAO header is set by default for files served as `application/ld+json` (among others) so that should be OK.

gosko committed:
Add redirect for hydra core/context.jsonld
